projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -253,17 +253,16 @@ const ProductCard = props => {
 				)
 			}
 		>
-			{
-				// If is not active, no reason to use children
-				// Since we want user to take action if isn't active
-				isActive && children ? (
-					children
-				) : (
-					<Text variant="body-small" className={ styles.description }>
-						{ description }
-					</Text>
-				)
-			}
+			<Text variant="body-small" className={ styles.description }>
+				{ description }
+			</Text>
+
+			{ isDataLoading ? (
+				<span className={ styles.loading }>{ __( 'Loading…', 'jetpack-my-jetpack' ) }</span>
+			) : (
+				children
```

Comment:
Should we still retain the `isActive` check here? I think the only other card that has children currently is the VideoPress card.

I don't think this is a blocker, just wondering if this impacts any other cards.

Reply:
As far as I can tell, this doesn't affect any cards since, like you said, they aren't using any children. The reasoning in the comment was that we don't want to show anything to the user unless it's active which is no longer true. 

Looking at the rest of the cards, it appears as though you are correct that only VideoPress uses children, and that card has its own check to not display them if VideoPress is not active so we are good there. I think we are good to remove this check